libraries/chain/proposal_evaluator.cpp

```diff
@@ -82,6 +82,11 @@ void_result proposal_create_evaluator::do_evaluate(const proposal_create_operati
    { // TODO: remove after HARDFORK_CORE_199_TIME has passed
       graphene::chain::impl::hf_199_visitor hf_199;
       hf_199( o );
```

Comment:
I think this won't compile.. Lack of a `}`.

Reply:
Sorry for that